templates/nextflow-ecs-cluster.yaml

```diff
@@ -67,6 +71,20 @@ Resources:
       ClusterSettings:
         - Name: containerInsights
           Value: enabled
+      CapacityProviders:
+        - !Ref EcsCapacityProvider
+
+  EcsCapacityProvider:
+    Type: AWS::ECS::CapacityProvider
+    Properties:
+      AutoScalingGroupProvider:
+        AutoScalingGroupArn: !Ref EcsAutoScalingGroup
+        ManagedScaling:
+          Status: ENABLED
+          MinimumScalingStepSize: 1
+          MaximumScalingStepSize: 1
+          TargetCapacity: 90
```

Comment:
The docs seem to indicate that if status is set to `enabled` then AWS auto scales for you.  I think that means that it will just ignore these scaling params?  or am i reading it wrong?..

```
The ManagedScaling property specifies the settings for the Auto
Scaling group capacity provider.

When managed scaling is enabled, Amazon ECS manages the scale-in
and scale-out actions of the Auto Scaling group. Amazon ECS manages a 
target tracking scaling policy using an Amazon ECS-managed CloudWatch 
metric with the specified targetCapacity value as the target value for the 
metric. For more information, see Using Managed Scaling in the Amazon 
Elastic Container Service Developer Guide.

If managed scaling is disabled, the user must manage the scaling of the 
Auto Scaling group. 
```

Reply:
@zaro0508 Good question. It's the opposite of how you're reading it; in my experience with clusters there's always some metric or metrics used as the measure so that it knows _when_ to autoscale. Now, TargetCapacity is not a required value so I assume there's some default value they're using, but they don't specify what it is. I went with 90 because the aws-samples code I was looking at used that value; it works well enough so I left it at that.

Look at this [set of instructions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/asg-capacity-providers-create-capacity-provider.html) for creating the capacity provider in the console -- there's more explanation at #9. You can also read the [deep dive blog post](https://aws.amazon.com/blogs/containers/deep-dive-on-amazon-ecs-cluster-auto-scaling/).